### PR TITLE
Test suggestions for #2071

### PR DESCRIPTION
**Pull request by srallen**
_Please request review from `@zooniverse/frontend` team. If PR is related to design, please request review from `@beckyrother` in addition._ 

Package: lib-classifier

These are suggested test improvement or additions for #2071 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?

## Components
- [ ] Has a storybook story been created or updated?
- [ ] Is the component accessible? 
  - [ ] Can it be used with a screen reader? [BBC guide to testing with VoiceOver](https://bbc.github.io/accessibility-news-and-you/accessibility-and-testing-with-voiceover-os.html)
  - [ ] Can it be used from the keyboard? [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - [ ] Is it passing accessibility checks in the storybook?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `yarn panic && yarn bootstrap` or `docker-compose up --build` and app works as expected?

## Publishing

- [ ] Is the changelog updated?
- [ ] Are the dependencies updated for apps and libraries that are using the newly published library?

## Post-merging

- [ ] Did the app deploy to https://frontend.preview.zooniverse.org/projects/:project-name/:owner or https://frontend.preview.zooniverse.org/about?
- [ ] Is the new feature working or bug now fixed?
  - [ ] Is there a Talk or blog post written to announce the new feature(s)?
- [ ] Is the design working across browsers (Firefox, Chrome, Edge, Safari) and mobile?
  - [ ] Is this approved by our designer?
- [ ] Is this ready for production deployment?

### CODE REVIEW

File: packages/lib-classifier/src/plugins/tasks/SurveyTask/components/SurveyTask.spec.js [hunk 1]
```diff
@@ -30,16 +30,21 @@ describe('SurveyTask', function () {
     expect(wrapper.find(Chooser)).to.have.lengthOf(1)
   })
 
-  describe.skip('with selectedChoice', function () {
+  describe('with selectedChoice', function () {
     before(function () {
-      wrapper = shallow(
+      sinon.stub(React, 'useRef').callsFake(() => { return { current: document.createElement('div') } })
```
Comment (mcbouslog) [inline packages/lib-classifier/src/plugins/tasks/SurveyTask/components/SurveyTask.spec.js:35]: Ah yes!! I forgot about `document.createElement('div')`. Thank you!

File: packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions.spec.js [hunk 2]
```diff
@@ -8,36 +8,38 @@ import Questions from './Questions'
...
       <Questions
         answers={{}}
-        questionIds={['HWMN', 'WHTBHVRSDS', 'RTHRNNGPRSNT']}
+        questionIds={questionIds}
         questions={mockTask.questions}
         setAnswers={setAnswersSpy}
       />
     )
+    checkboxes = wrapper.find(CheckBoxGroup)
+    radioButtons = wrapper.find(RadioButtonGroup)
```
Comment (mcbouslog) [inline packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Choice/components/Questions.spec.js:24]: Much cleaner!

File: packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.spec.js [hunk 3]
```diff
@@ -9,6 +9,7 @@ import { default as Task } from '@plugins/tasks/SurveyTask'
 import FilterStatus from './FilterStatus'
 import Characteristics from '../Characteristics'
 import FilterButton from '../Characteristics/components/FilterButton'
+import { expect } from 'chai'
```
Comment (mcbouslog) [inline packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.spec.js:12]: I think VSCode autocompletes/autoinserts this?

Comment (srallen) [reply packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/CharacteristicsFilter/FilterStatus/FilterStatus.spec.js:12]: It does. I didn't catch that it had here.

**Pull request merged**